pkg/apis/eksctl.io/v1alpha3/zz_generated.deepcopy.go

```diff
@@ -137,6 +137,27 @@ func (in *ClusterMeta) DeepCopy() *ClusterMeta {
 	return out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *ClusterStatus) DeepCopyInto(out *ClusterStatus) {
+	*out = *in
+	if in.CertificateAuthorityData != nil {
+		in, out := &in.CertificateAuthorityData, &out.CertificateAuthorityData
```

Comment:
it seems that you wouldn't need to assign the addresses of slices here since you dereference their use below

Reply:
Also, `deepcopy-gen`, so no need for us to worry.